test/suite/test_hs11.py

```diff
@@ -44,8 +44,18 @@ class test_hs11(wttest.WiredTigerTestCase):
         ('deletion', dict(update_type='deletion')),
         ('update', dict(update_type='update'))
     ]
-    scenarios = make_scenarios(format_values, update_type_values)
+    long_running_txn_values = [
+       ('long-running', dict(long_run_txn=True)),
```

Comment:
This test doesn't pass until WT-10512 is merged. we need to decide the merge order to enable it.

Reply:
If we merge that first, this test should pass.